README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img src="docs/logo.png" height="200"><br>
+  <img src="imgs/logo.png" height="200"><br>
```

Comment:
Let`s move the logo to the docs/src/assets folder as in https://github.com/JuliaEarth/GeoStats.jl/blob/master/docs/src/assets/logo.svg

Reply:
OK

Reply:
Made in [44a7d1b](https://github.com/JuliaML/TableTransforms.jl/pull/46/commits/44a7d1b3391c77a08f08f9b597d00bc07eb40b97)